tests/MongoDB.Driver.Tests/Specifications/transactions-convenient-api/tests/callback-aborts.yml

```diff
@@ -1,3 +1,11 @@
+runOn:
```

Comment:
Do we really need to have `yml` files? Maybe we can remove them? Just as a suggestion.

Reply:
You are right that the yml files are technically not needed.

But a good reason to have them is that it makes it easier to diff the specifications repo tests directory with our tests directory. If we didn't have the yml files they wouldn't be in perfect sync.

Also... sometimes the yml files have interesting comments in them and it can be convenient to have them locally rather than having to go to the specifications repo to look at them.

Reply:
Got it